website/content/en/docs/deployment/cognito-rds-s3/guide-terraform.md

```diff
@@ -99,9 +97,9 @@ pwd
     EOF
     ```
 
-### All Configurations
+### View all Configurations
 
-A full list of inputs for the terraform stack can be found [here](https://github.com/awslabs/kubeflow-manifests/blob/main/deployments/cognito-rds-s3/terraform/variables.tf).
+View all possible configuration options in this [full list of inputs for the terraform stack](https://github.com/awslabs/kubeflow-manifests/blob/main/deployments/cognito-rds-s3/terraform/variables.tf).
```

Comment:
View all possible configuration options in variables.tf file of the terraform stack sounds better?